extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientBuildItem.java

```diff
@@ -1,26 +1,12 @@
 package io.quarkus.mongodb.deployment;
 
-import com.mongodb.client.MongoClient;
-
 import io.quarkus.builder.item.SimpleBuildItem;
-import io.quarkus.mongodb.ReactiveMongoClient;
-import io.quarkus.runtime.RuntimeValue;
 
+/**
+ * Marker build item indicating the MongoClient has been fully initialized.
+ */
 public final class MongoClientBuildItem extends SimpleBuildItem {
```

Comment:
Unless I am missing something, it doesn't seem like this is being consumed anywhere?